spec/features/signup_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+feature "User signs up" do
+
+  scenario 'when signing up I should change user count' do
+    expect { sign_up }.to change(User, :count).by(1)
+    expect(User.first.email).to eq("alice@example.com")
+    expect(User.first.name).to eq("alice")
+  end
+
+  scenario 'after signing up User should see custom greeting' do
+    sign_up
+    expect(page).to have_content("Welcome, alice101")
+  end
```

Comment:
You are currently showing the email instead of the username, so this test is failing